### DISCUSSION TITLE
Sounds

### DISCUSSION THREAD

**Discussion by thelift112** [Jcw87/c2-sans-fight#119] [General]
I found some really, really interesting, if you set sound's pitch to 0.1 you get some interesting results... such as PlayerFight.ogg becomes a laugh and Warning.ogg becomes something that sounds a bit like Gaster... I may update this list sometime.
PlayerFight.ogg - laugh
Warning.ogg - Gaster sounding

**Comment by Nightingale08** [comment]
Hmmmm where can i find the sounds folder? Or all the possible sounds you can play with the function "Sound"?

**Reply by thelift112** [reply]
You can find them all right here: https://github.com/Jcw87/c2-sans-fight/tree/master/Files and you can add pitch by doing 0,Sound,(sound),(pitch)
